File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x < 0)
        {
            return false;
        }

        long rev = 0;
        int temp=0;
        temp = x;
        while(temp != 0)
        {
            int rem=0;
            rem = temp % 10;
            rev = rev*10 + rem;
            temp = temp /10;

        }
        if(rev == x)
        {
            return true;
        }else{
            return false;
        }


    }
};

File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int reverse=0;
        if(x<0)
        {
            return false;
        }
        for(int i=x ;i>0;i=i/10)
        {
            reverse = reverse*10+(i%10);
        }

        if(x == reverse)
        {
            return true;
        }
        else{
            return false;
        }
       

        
    }
}